lib/logitech_receiver/descriptors.py

```diff
@@ -208,7 +208,7 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 # Mice
 
 _D('Wireless Mouse M175', protocol=2.0, wpid='4008')
-_D('Wireless Mouse M185')
+_D('Wireless Mouse M185', protocol=2.0, wpid='4054')
```

Comment:
Same here, my M185 is also HID++ 2.0 but since M185 descriptor was already present, maybe there is a version with HID++ 1.0 out there. If so, add protocol 2.0 will break compatibility.